src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -95,8 +95,8 @@ trait EntityComponent {
       } else {
         // slick can't do a query with '(entityType, entityName) in ((?, ?), (?, ?), ...)' so we need raw sql
         val baseSelect = sql"select id, name, entity_type, workspace_id, record_version from ENTITY where workspace_id = $workspaceId and (entity_type, name) in ("
-        val entityTypeNameTuples = entities.map { case entity => sql"(${entity.entityType}, ${entity.entityName})" }.reduce((a, b) => concatSqlActionsWithDelim(a, b, sql","))
-        concatSqlActions(concatSqlActions(baseSelect, entityTypeNameTuples), sql")").as[EntityRecord]
+        val entityTypeNameTuples = reduceSqlActionsWithDelim(sql",", entities.map { case entity => sql"(${entity.entityType}, ${entity.entityName})" }.toSeq)
```

Comment:
make it a default parameter? I still think it is a good idea to make it a parameter.